sdk/servicebus/Azure.Messaging.ServiceBus/CHANGELOG.md

```diff
@@ -2,8 +2,16 @@
 
 ## 7.16.0-beta.1 (Unreleased)
 
+### Acknowledgments
+Thank you to our developer community members who helped to make the Service Bus client library better with their contributions to this release:
+
+- Daniel Marbach  _([GitHub](https://github.com/danielmarbach))_
+
 ### Features Added
 
+- `ProcessMessageEventArgs` provides a `MessageLockCancellationToken` that gets cancelled when the `ServiceBusReceivedMessage.LockUntil` time expired.
```

Comment:
nit: can we also mention that it will be canceled when the processor encounters an exception indicating the lock is lost?